### PR TITLE
add debian-trixie build target

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Comment by Habbie** [comment]
@zeha you may want to cherrypick the sdig commit to your tree. It turns out that `dig @127.0.0.1 ...` can error on parsing `resolv.conf` if it does not like the content.

File: builder-support/debian/authoritative/debian-buster/tests/control [hunk 1]
```diff
@@ -1,25 +1,29 @@
 Tests: smoke-bind
 Depends: dnsutils,
+         pdns-tools,
          pdns-backend-bind,
          pdns-server
 Restrictions: needs-root
 
 Tests: smoke-mysql
 Depends: dnsutils,
```
Comment (zeha) [inline builder-support/debian/authoritative/debian-buster/tests/control:9]: i tried to keep the dependencies sorted by name

Comment (Habbie) [reply builder-support/debian/authoritative/debian-buster/tests/control:9]: makes sense!

Comment (Habbie) [reply builder-support/debian/authoritative/debian-buster/tests/control:9]: oh, do we even still need dnsutils?

Comment (zeha) [reply builder-support/debian/authoritative/debian-buster/tests/control:9]: probably not!

Comment (Habbie) [reply builder-support/debian/authoritative/debian-buster/tests/control:9]: ok, will test without

Comment (Habbie) [reply builder-support/debian/authoritative/debian-buster/tests/control]: PR amended!

**Comment by Habbie** [comment]
test builds at

* auth: https://github.com/Habbie/pdns/actions/runs/5448391969
* rec: https://github.com/Habbie/pdns/actions/runs/5448393008
* dnsdist: https://github.com/Habbie/pdns/actions/runs/5448393886

**Comment by zeha** [comment]
I've uploaded this to Debian unstable

**Review by rgacogne — APPROVED** [review]

**Pull request merged**